### PR TITLE
add remove-AzureRmFrontDoorContent cmdlet

### PR DESCRIPTION
**Pull request by hytao**
## Description

Add Remove-AzureRmFrontDoorContent cmdlet
Update input parameters in New-AzureRmFrontDoorFirewallPolicy
Design review: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/184
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/FrontDoor/Commands.FrontDoor/help/Remove-AzureRmFrontDoorContent.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+external help file: Microsoft.Azure.Commands.FrontDoor.dll-Help.xml
+Module Name: AzureRM.FrontDoor
+online version:
```
Comment (maddieclayton) [inline src/ResourceManager/FrontDoor/Commands.FrontDoor/help/Remove-AzureRmFrontDoorContent.md]: Follow instructions here to fix CI: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell/8440/artifact/src/Package/ValidateHelpIssues.csv/*view*/.

File: src/ResourceManager/FrontDoor/Commands.FrontDoor/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -25,5 +25,5 @@
 [assembly: ComVisible(false)]
 [assembly: CLSCompliant(false)]
 [assembly: Guid("d90791a2-8102-47fc-2483-de25ae796eb1")]
-[assembly: AssemblyVersion("0.1.0")]
-[assembly: AssemblyFileVersion("0.1.0")]
+[assembly: AssemblyVersion("0.2.0")]
```
Comment (maddieclayton) [inline src/ResourceManager/FrontDoor/Commands.FrontDoor/Properties/AssemblyInfo.cs]: revert this, I'll take care of it later.

File: src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoorContent.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        /// </summary>
+        [Parameter(Mandatory = true, HelpMessage = "The paths to the content to be purged.")]
+        public string[] ContentPath { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            if (ShouldProcess(Resources.FrontDoorTarget, string.Format(Resources.PurgeFrontDoor, Name)))
+            {
+                var parameters = new Microsoft.Azure.Management.FrontDoor.Models.PurgeParameters(ContentPath);
+                FrontDoorManagementClient.Endpoints.PurgeContent(ResourceGroupName, Name, parameters);
```
Comment (maddieclayton) [inline src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoorContent.cs]: If you give an outputtype above, you must return something in the cmdlet.

File: src/ResourceManager/FrontDoor/Commands.FrontDoor/ChangeLog.md [hunk 4]
```diff
@@ -19,5 +19,8 @@
 -->
 ## Current Release
 
+## Version 0.2.0
+* Add remove-AzureRmFrontDoorContent for cache purge
```
Comment (maddieclayton) [inline src/ResourceManager/FrontDoor/Commands.FrontDoor/ChangeLog.md]: Just put this directly underneath "Current Release"

File: src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoorContent.cs [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Microsoft.Azure.Management.FrontDoor;
+using System.Linq;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+namespace Microsoft.Azure.Commands.FrontDoor.Cmdlets
+{
+    /// <summary>
+    /// Defines the New-AzureRmFrontDoor cmdlet.
+    /// </summary>
+    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FrontDoorContent", SupportsShouldProcess = true), OutputType(typeof(void))]
```
Comment (maddieclayton) [inline src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoorContent.cs]: Please follow the same convention for the output as this: https://github.com/hytao/azure-powershell/blob/frontdoor/src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoor.cs

File: src/ResourceManager/FrontDoor/Commands.FrontDoor/help/Remove-AzureRmFrontDoorContent.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### None
+
+## OUTPUTS
+
+### Microsoft.Azure.Commands.FrontDoor.Models.PSFrontDoor
```
Comment (maddieclayton) [inline src/ResourceManager/FrontDoor/Commands.FrontDoor/help/Remove-AzureRmFrontDoorContent.md]: Change this manually to System.Boolean (it doesn't update automatically.

File: src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoorContent.cs (hunk 5 above)
Comment (hytao) [reply src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoorContent.cs]: Do I need to add -PassThru for this?

Comment (maddieclayton) [reply src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoorContent.cs]: Yes

Comment (hytao) [reply src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoorContent.cs]: Added

**Comment by maddieclayton** [comment]
Will create a versioning PR for this change in AzureRM and Az.

**Pull request merged**